### PR TITLE
fix(card): integrate elevation

### PR DESCRIPTION
**Pull request by rachelbt**
updated the style as in vivid-3:
https://github.com/Vonage/vivid-3/pull/268

added hight property to match the elevation inside card

### CODE REVIEW

File: components/card/src/vwc-card.ts [hunk 1]
```diff
@@ -7,14 +7,17 @@ import { property } from 'lit-element/lib/decorators.js';
...
-
+import '@vonage/vwc-elevation';
 
 declare global {
 	interface HTMLElementTagNameMap {
 		'vwc-card': VWCCard;
 	}
 }
 
+const elevationSets = ['0' , '2', '4', '8', '12' , '16' , '24'];
```
Comment (yinonov) [inline components/card/src/vwc-card.ts:18]: can be imported from elevation

**Review by yinonov — CHANGES_REQUESTED** [review]

File: components/card/src/vwc-card.ts (hunk 1 above)
Comment (rachelbt) [reply components/card/src/vwc-card.ts:18]: there are non in the elevation comp... 😨

Comment (yinonov) [reply components/card/src/vwc-card.ts:18]: we should open a ticket to support ts types of elevation tokens

File: ui-tests/tests/vwc-card/index.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ export async function createElementVariations(wrapper) {
 			<div slot="content">I'm the card content that overrides the content from attributes. I have no text styles or any styles for that metter :) You can do whatever you want here</div>
 		</vwc-card>
 
-		<vwc-card >
+		<vwc-card dp="24">
```
Comment (yinonov) [inline ui-tests/tests/vwc-card/index.js:100]: Elevation u mean?

**Review by yinonov — CHANGES_REQUESTED** [review]

File: ui-tests/tests/vwc-card/index.js (hunk 2 above)
Comment (rachelbt) [reply ui-tests/tests/vwc-card/index.js:100]: Its a test for the dp change and for the content slot

Comment (yinonov) [reply ui-tests/tests/vwc-card/index.js:100]: the _card_ api member should be `elevation`. in opposed to _elevation_ where its member is, indeed, `dp`

Comment (rachelbt) [reply ui-tests/tests/vwc-card/index.js]: ohhhhhh. Got it.
Fixed it

**Review by yinonov — APPROVED** [review]

**Pull request merged**